sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -403,6 +450,18 @@ object AdaptiveSparkPlanExec {
   private val executionContext = ExecutionContext.fromExecutorService(
     ThreadUtils.newDaemonCachedThreadPool("QueryStageCreator", 16))
 
+  /**
+   * The temporary [[LogicalPlan]] link for query stages.
+   *
+   * Temp logical links are set for each query stage after its creation. During re-planning, the
```

Comment:
but what I see from the code is, we set the normal logical link after query stage is created: https://github.com/apache/spark/pull/25456/files#diff-6954dd8020a9ca298f1fb9602c0e831cR291

Reply:
We could do either way (temp or normal link) here I think, because we have a post-fix for the link in https://github.com/apache/spark/pull/25456/files#diff-6954dd8020a9ca298f1fb9602c0e831cR362, which by itself is necessary.

Reply:
let's make sure the comment and the code are consistent.